### PR TITLE
1) created Ball class with the following fields: color and number and…

### PR DESCRIPTION
**Pull request by Alex-Solod**
… override toString() method.

2) created enum with different colors
3) implemented method getRandomColor() in class ColorSupplier that return a random color, based on the enum values by using Random class. 
4) created a Lottery class with getRandomBall() method which return the ball with a random color  and a random number (the maximum number: 100) 
5) to test the program, in main() method, I created three balls using getRandomBall() method from Lottery class and print information about them in the console.

### CODE REVIEW

File: src/main/java/core/basesyntax/Colors.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Colors.java:3]:
```suggestion
public enum Color {
```

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/ColorSupplier.java:6]: image

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 3]
```diff
@@ -1,7 +1,12 @@
...
+
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: remove empty line

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
+        Random random = new Random();
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/ColorSupplier.java:8]: move Random to the class field, like: 
image

File: src/main/java/core/basesyntax/Lottery.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:7]: remove empty line

File: src/main/java/core/basesyntax/Lottery.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:6]: remove static

File: src/main/java/core/basesyntax/Lottery.java [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+        Colors color = ColorSupplier.getRandomColor();
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:8]: move Random and ColorSupplier to the class variables

File: src/main/java/core/basesyntax/Lottery.java [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+        int number = new Random().nextInt(100) + 1;
```
Comment (kozhukhovsky) [inline src/main/java/core/basesyntax/Lottery.java:9]: 100 is magic number
image

**Review by kozhukhovsky — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Lottery.java (hunk 6 above)
Comment (Alex-Solod) [reply src/main/java/core/basesyntax/Lottery.java:6]: After removing static from Ball getRandomBall() method the program does not want to compile and show (non-static method getRandomBall() cannot be referenced from a static context) in Main method (Ball ball1 = Lottery.getRandomBall(); )

File: src/main/java/core/basesyntax/ColorSupplier.java (hunk 2 above)
Comment (Alex-Solod) [reply src/main/java/core/basesyntax/ColorSupplier.java:6]: After removing static from Color getRandomColor() method the program does not want to compile and show (non-static method getRandomColor() cannot be referenced from a static context) in Main method (Ball ball1 = Lottery.getRandomBall(); ) 
I don't know how to fix that

**Comment by okuzan** [comment]
Don't request a review until you fixed your build, if you can't - ask for help in the chat.
image

**Review by okuzan — CHANGES_REQUESTED** [review]
see comment

**Pull request closed without merging**